PAS_Database/dbo/Stored Procedures/SP_GetUnitSalePriceHistoryById.sql

```diff
@@ -0,0 +1,58 @@
+﻿-- =============================================
+-- Author:		Ekta Chandegra
+-- Create date: 09-01-2024
+-- Description:	This stored procedure is used to retrieve history of unit sales price.
+-- =============================================
+
+/*************************************************************   
+ **************************************************************           
+  ** Change History           
+ **************************************************************           
+ ** PR   Date         Author				Change Description            
+ ** --   --------     -------				--------------------------------          
+    1    09-01-2024   Ekta Chandegra		Created
+
+	EXEC [GetUnitSalePriceHistoryById] 163178
+
+**************************************************************/
+
+CREATE   PROC [dbo].[SP_GetUnitSalePriceHistoryById]
+@StocklineId bigint
+
+
+AS
+BEGIN
+    SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED
+	SET NOCOUNT ON;
+	BEGIN TRY
+	BEGIN TRANSACTION
+	BEGIN
+		
+		SELECT StockLineNumber,UnitSalesPrice,SalesPriceExpiryDate,CreatedDate,
+		UpdatedDate,UpdatedBy
+		FROM [dbo].[UnitSalePriceHistoryAudit] 
```

Comment:
No lock is missing and no need of using transaction when only select statements are there